samples/drivers/CAN/mcp2515-dts.overlay

```diff
@@ -6,13 +6,15 @@
 
 &spi1 {
 	status = "ok";
+	cs-gpios = <&gpioa 4 GPIO_DIR_OUT>;
 	mcp2515@0 {
 		compatible = "microchip,mcp2515";
```

Comment:
If we give it a label properties can be changed in additional (e.g. application specific) overlay files.

```dts
&spi1 {
	/* ... */
        mcp2515_can1: mcp2515@0 {
		/* ... */
	};
};

/* in different file, overlay or position: */
&mcp2515_can1 {
	bus-speed = <250000>;
};
```